addons/point_of_sale/static/src/js/Screens/OrderManagementScreen/ControlButtons/ReprintReceiptButton.js

```diff
@@ -19,18 +18,7 @@ odoo.define('point_of_sale.ReprintReceiptButton', function (require) {
             const order = this.orderManagementContext.selectedOrder;
             if (!order) return;
 
-            if (this.env.pos.proxy.printer) {
-                const fixture = document.createElement('div');
-                const orderReceipt = new (Registries.Component.get(OrderReceipt))(this, { order });
-                await orderReceipt.mount(fixture);
-                const receiptHtml = orderReceipt.el.outerHTML;
-                const printResult = await this.env.pos.proxy.printer.print_receipt(receiptHtml);
-                if (!printResult.successful) {
-                    this.showTempScreen('ReprintReceiptScreen', { order: order });
-                }
-            } else {
-                this.showTempScreen('ReprintReceiptScreen', { order: order });
-            }
+            this.showTempScreen('ReprintReceiptScreen', { order: order });
```

Comment:
why did you do this commit?

Reply:
@rhe-odoo can you just remind me why this commit is needed? and also name it FIX if it is a fix

Reply:
This is because the inheritance was not working.
Discussed with JCB, we figured out that all the stuff was not needed.